src/System.Runtime/tests/System/Reflection/TypeInfoTests.netcoreapp.cs

```diff
@@ -17,7 +17,6 @@ private static IEnumerable<object[]> SZArrayOrNotTypes()
             yield return new object[] { typeof(int), false };
             yield return new object[] { typeof(int[]).MakeByRefType(), false };
             yield return new object[] { typeof(int[,]), false };
-            yield return new object[] { typeof(TypeInfoTests), false };
```

Comment:
Should this rather be changed to `typeof(TypeInfoNetcoreTests)` instead of deleting?

Reply:
Brought back related data case